index.html

```diff
@@ -140,16 +140,17 @@ <h2>About me</h2>
       <p>
         I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
       </p>
-      <form action="https://formspree.io/f/xbjezgwz" method="POST">
+      <form action="https://formspree.io/f/xbjezgwz" method="POST" id="contactForm">
         <fieldset>
           <label>
-            <input type="text" name="user_name" placeholder="Full name" required>
+            <input id="name" type="text" name="user_name" placeholder="Full name" required>
           </label>
           <label>
-            <input type="email" name="user_email" placeholder="Email" required>
+            <input id="email" type="email" name="user_email" placeholder="Email" required>
```

Comment:
- [OPTIONAL] It will be better if before submitting the contact form and check if email field containing uppercase characters to check first the email type attribute validation on the email input field in order to prevent this 👇🏻 

![image](https://user-images.githubusercontent.com/79058364/233491521-ccdb1d6c-39a7-4dfb-808c-b63c1f45bbce.png)